content/docs/add-react-to-a-website.md

```diff
@@ -8,195 +8,194 @@ prev: getting-started.html
+De meerderheid van website zijn niet, en hoeven geen Single Page Applicatie te zijn. Met **een paar regels code en zonder build tooling**, kun je React uitproberen op een klein deel van je website. Je kunt ofwel langzaamaan de aanwezigheid uitbreiden, of het beperkt houden tot een paar dynamische widgets.
 
 ---
 
-- [Add React in One Minute](#add-react-in-one-minute)
-- [Optional: Try React with JSX](#optional-try-react-with-jsx) (no bundler necessary!)
+- [React toevoegen binnen een minuut](#add-react-in-one-minute)
+- [Optioneel: Probeer React met JSX](#optional-try-react-with-jsx) (no bundler necessary!)
 
-## Add React in One Minute {#add-react-in-one-minute}
+## React toevoegen binnen een minuut {#add-react-in-one-minute}
 
-In this section, we will show how to add a React component to an existing HTML page. You can follow along with your own website, or create an empty HTML file to practice.
+In deze sectie laten we zien hoe je een React component kunt toevoegen aan een bestaande HTML pagina. Je kunt dit deel volgen met je eigen website, of een leeg HTML bestande aanmaken om te oefenen.
 
-There will be no complicated tools or install requirements -- **to complete this section, you only need an internet connection, and a minute of your time.**
+We hebben geen gecompliceerde tools of installaties nodig. -- **om deze sectie te voltooien, heb je alleen een internet verbinding en een minuutje van je tijd nodig**.
 
-Optional: [Download the full example (2KB zipped)](https://gist.github.com/gaearon/6668a1f6986742109c00a581ce704605/archive/f6c882b6ae18bde42dcf6fdb751aae93495a2275.zip)
+Optioneel: [Download het volledige voorbeeld (2KB zipped)](https://gist.github.com/gaearon/6668a1f6986742109c00a581ce704605/archive/f6c882b6ae18bde42dcf6fdb751aae93495a2275.zip)
 
-### Step 1: Add a DOM Container to the HTML {#step-1-add-a-dom-container-to-the-html}
+### Stap 1: Voeg een DOM Container toe aan de HTML {#step-1-add-a-dom-container-to-the-html}
 
-First, open the HTML page you want to edit. Add an empty `<div>` tag to mark the spot where you want to display something with React. For example:
+Start met het openen van de HTML pagina die je wilt bewerken. Voeg een lege `<div>` tag toe om aan te geven waar je iets met React wilt weergeven. Bijvoorbeeld:
 
 ```html{3}
-<!-- ... existing HTML ... -->
+<!-- ... bestaande HTML ... -->
 
 <div id="like_button_container"></div>
 
-<!-- ... existing HTML ... -->
+<!-- ... bestaande HTML ... -->
 ```
 
-We gave this `<div>` a unique `id` HTML attribute. This will allow us to find it from the JavaScript code later and display a React component inside of it.
+We hebben deze `<div>` een uniek `id` HTML attribuut. Hiermee kunnen het het element later met JavaScript vinden en een React component er in tonen.
```

Comment:
het het element => het element
en een React component er in tonen = en er een React component in tonen